wbooks/src/redux/suggestionAction/reducer.js

```diff
@@ -0,0 +1,23 @@
+import { actionNames } from './actions';
+
+const initialState = {
+  suggestionList: []
+};
+
+const suggestionAction = (state = initialState, action) => {
+  switch (action.type) {
+    case actionNames.GET_SUGGESTION_SUCCESFULL:
+      return {
+        ...state,
+        suggestionList: action.payload.suggestions.slice(0, 4)
+      };
+    case actionNames.GET_SUGGESTION_FAILURE:
+      return {
+        state
+      };
```

Comment:
Be aware of that ! This will return a `{ state: state }` object instead of `state`.
If this action doesn't modify the state, this shouldn't belong here or exist.